content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -293,6 +293,10 @@ Services.
 Readiness probes runs on the container during its whole lifecycle.
 {{< /note >}}
 
+{{< warning >}}
+Liveness probes *do not* wait for readiness probes to succeed. If you want to wait before executing a liveness probe you should use initialDelaySeconds or a startupProbe.
```

Comment:
How about this revised wording:
```suggestion
Liveness probes become active as soon as the container is started: they do *not* wait
for readiness probes to succeed. If you want to wait before executing a liveness
probe, you can set `initialDelaySeconds`, or use a startupProbe.
```

Reply:
Also, I'd probably use a _caution_ callout rather than a _warning_; _warning_ is for “a mistake here could break your cluster” sort of severity.

Reply:
+1 to tim's comments and revision :)

Reply:
@caleb15 .
Do you need help or have questions about the suggested changes?

Reply:
Sorry about that, I had slack notifications enabled but they don't work for repos that don't have the slack app installed. I just enabled re-enabled email notifications.

The change looks good. I'll note that it should be incorporated into the change for https://github.com/kubernetes/website/issues/26785 now.